docs/apis-tools/migration-manuals/migrate-to-zeebe-user-tasks.md

```diff
@@ -181,176 +167,183 @@ We recommend you migrate process-by-process, allowing you to thoroughly test the
 
 Repeat these steps for all user tasks in the process. Then, deploy the process to your development cluster and test it by running the process and ensuring your custom task applications work.
 
-## Use the new Zeebe Task API
+## Use Camunda 8 API
 
 :::note
 The Tasklist REST API is not deprecated, and you still need it for queries on both task types.
```

Comment:
This note won't be quite right in 8.7. The Tasklist REST API will be marked as deprecated, but not removed until 8.9. Will this be adjusted in a future PR? Internal context [here](https://docs.google.com/document/d/1aqwSXP8uF3I7cV6Rz-HmBYIXSltwlmfjHKmmlVWBg6o/edit?tab=t.0#heading=h.zhyuwlliwpqj).

Reply:
To adjust with https://github.com/camunda/camunda-docs/pull/4723 👍